lib/api/interceptor.js

```diff
@@ -22,20 +22,21 @@ class XHRInterceptor {
 
       const mockedRequests = Scenarios.getCurrentMockedRequests();
       for (let mockedRequest of mockedRequests) {
-        const methodsMatch = request.method === mockedRequest.method;
-        const urlsMatch    = request.url === mockedRequest.url;
-        const paramsMatch  = request.body === mockedRequest.params;
-
-        if (methodsMatch && urlsMatch && paramsMatch && mockedRequest.active) {
-          return this.buildResponse(responder, mockedRequest.getResponse(mockedRequest.selectedStateId), request);
+        if (mockedRequest.matches({
```

Comment:
Actually, the `matches` function also receives a `Request` model
Plus, in and `xhrRequest` it's called `body`, and for some reason in request you called it `params`